index.html

```diff
@@ -61,5 +61,7 @@ <h1>Tasks</h1>
     <button class="actionButton" aria-label="Add new task">
       <img src="../../assets/plusicon.svg" alt="Add Icon" />
     </button>
+
+    <script src="index.js" defer="true"></script>
```

Comment:
No value required (following MDN)

```suggestion
    <script src="index.js" defer></script>
```